content/docs/getting-started/README.md

```diff
@@ -49,33 +51,48 @@ You will also need to install the following software on your laptop:
 
 > ℹ️ Try running `gcloud components install kubectl` to quickly install `kubectl`.
 
-This tutorial requires you to run certain commands and you will need to adapt some of them for your environment.
-Where ever you see a `$VARIABLE_NAME` in a command, you need to either replace the variable before you execute the command,
-or assign a value to the variable and export it so that the value can be substituted automatically by your shell.
+## 0. Configure `gcloud` with a Google Cloud project
+
+If you don't have a Google Cloud account, do not worry, the command below will create it for you:
 
 ```bash
-export PROJECT="..."       # Your Google Cloud project ID e.g. jetstack-richard
-export REGION="..."        # Your Google Cloud region e.g. europe-west1
-export CLUSTER="..."       # A name for your GKE cluster e.g. test-cluster-1
-export ZONE="..."          # The Google Cloud DNS zone in which to create your website domain name e.g. example-zone
-export DOMAIN_NAME="..."   # The domain name for your website e.g. www.example.com
-export EMAIL_ADDRESS="..." # The email address to register with Let's Encrypt e.g. devops@example.com
```

Comment:
These upfront environment variables put me off, since it seems I am meant to set the all up right now but I have no idea what "zone" is and what "cluster name" is and what "email address" is